dist/_patterns/01-components/01-page-elements/hero.twig

```diff
@@ -0,0 +1,10 @@
+<div class="hero">
+  <div class="hero__content" role=”banner”>
+    <h1 class="hero__header">{{ pageTitle }}</h1>
+    <h2 class="hero__subheader">{{ pageSubTitle }}</h2>
+    <p class="hero__text">{{ summary }}</p>
```

Comment:
I think we should use <div> instead of <p> since the summary can contain HTML, and may come from a WYSIWYG field in Drupal.

We should also wrap this in an if statement so that the wrapper markup isn’t output if there’s nothing in the {{ summary }} variable.